openlibrary/templates/books/custom_carousel.html

```diff
@@ -0,0 +1,38 @@
+$def with(books=None, title="", url="", key="", min_books=6, cta="Borrow", test=False)
+
+$if test or (books and len(books) >= min_books):
+    <div class="carousel-section">
+      <div class="carousel-section-header">
+        <h2 class="home-h2"><a href="$url">$title</a></h2>
+      </div>
+      <div class="carousel-container carousel-container-decorated">
+        <div class="carousel carousel-$key">
+          $for book in (books or []):
+            $ byline = (' by ' + ', '.join([author.name for author in book.authors])) if book.get('authors') else ''
```

Comment:
I assume there's no significant performance penalty in calling `book.authors` twice? Not a network call, right?

Reply:
that's right, book is an obj which is already fetched, it's just an access of the dict